packages/headless/src/controllers/field-suggestions/category-facet/headless-category-field-suggestions.ts

```diff
@@ -213,6 +134,10 @@ export interface CategoryFieldSuggestionsProps {
 
 /**
  * Creates a `CategoryFieldSuggestions` controller instance.
+ *
+ * This controller initializes a category facet, but exposes state and methods that are relevant for suggesting fields values based on a query.
+ * It's important not to initialize a category facet with the same `facetId` but different options, because only the options of the controller which is built first will be taken.
```

Comment:
```suggestion
 * This controller initializes a category facet under the hood, but exposes state and methods that are relevant for suggesting field values based on a query.
 * It's important not to initialize a category facet with the same `facetId` but different options, because only the options of the controller which is built first will be taken into account.
```